linear_search/Python/linear_search.py

```diff
@@ -0,0 +1,10 @@
+def linear_search(values, search_for):
+    search_at = 0
+    search_res = False
+    while search_at < len(values) and search_res is False:
+        if values[search_at] == search_for:
+            search_res = True
+        else:
+            search_at = search_at + 1
+
+    return search_res
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpsouua9a1/linear_search/Python/linear_search.py
+++ b/tmp/tmpsouua9a1/linear_search/Python/linear_search.py
@@ -7,4 +7,4 @@
         else:
             search_at = search_at + 1
 
-    return search_res+    return search_res
```